Library/Homebrew/deprecate_disable.rb

```diff
@@ -27,6 +27,8 @@ module DeprecateDisable
     no_longer_available:      "is no longer available upstream",
     no_longer_meets_criteria: "no longer meets the criteria for acceptable casks",
     unmaintained:             "is not maintained upstream",
+    fails_gatekeeper_check:   "does not pass macOS Gatekeeper check",
+    # TODO: remove after all casks are updated to use this symbol
     unsigned:                 "is unsigned or does not meet signature requirements",
```

Comment:
What is the standard procedure here? Do we need to keep around for some time, for anyone who has an old cask definition around, or can we just remove it after `homebrew-cask` has been updated?

Reply:
@bevanjkay We have to keep it around indefinitely but use `odeprecate` etc. Can you add the string `odeprecate` in the comment somewhere so it gets caught for next major/minor release?

Reply:
Makes sense.